documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>I'm a bit unsure about the "making them a great entry point to tensorflow and ML" fragment. It's a little editorial, but OTOH, the tone is probably fine for an intro / beginner's guide, so I think leave it in.</p><p> </p><p>Change "tensorflow" to "TensorFlow" though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>